src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,141 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.common.not-implemented.view :as not-implemented]
+            [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [quo2.foundations.resources :as resources]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [utils.i18n :as i18n]))
+
+(defn- left-loading
+  [{:keys [size blur? theme]}]
+  [rn/view {:style (style/loading-container size blur? theme)}])
+
+(defn- left-subtitle
+  [{:keys [theme size description icon icon-color blur? subtitle emoji-color emoji]}]
+  (let [background-color (colors/theme-colors
+                          (colors/custom-color emoji-color 50)
+                          (colors/custom-color emoji-color 60)
+                          theme)]
+    [rn/view {:style style/subtitle-container}
+     (when (not= :small size)
+       [rn/view {:style (style/subtitle-icon-container description)}
+        (case description
+          :icon    [icons/icon icon
+                    {:accessibility-label :description-icon
+                     :size                16
+                     :color               icon-color}]
+          :account [account-avatar/view
+                    {:customization-color background-color
```

Comment:
@mmilad75 - Please rename `emoji-color` to `customization-color` (_or `account-customization-color` if the component has `customization-color` key already_), and the `background-color` var. 

Pass the `customization-color` directly to the `account-avatar` component.  It already takes care of the `background-color` based on `customization-color` 👍

Reply:
yes, here we should be passing `customization-color` prop to the next component. We don't need to do the `colors/theme-colors` resolving.
i.e
```
(defn- left-subtitle
  [{:keys [theme size description icon icon-color blur? subtitle customization-color emoji]}]

    [rn/view {:style style/subtitle-container}
     (when (not= :small size)
       [rn/view {:style (style/subtitle-icon-container description)}
        (case description
          :icon    [icons/icon icon
                    {:accessibility-label :description-icon
                     :size                16
                     :color               icon-color}]
          :account [account-avatar/view
                    {:customization-color customization-color
```
and as @smohamedjavid  said we don't want to name this prop anything other than `customization-color` as we have it this way in all other components 👍

Reply:
Done